data/src/main/java/camp/nextstep/edu/github/GitHubInfoResponse.kt

```diff
@@ -0,0 +1,27 @@
+package camp.nextstep.edu.github
+
+import com.google.gson.annotations.SerializedName
+
+internal data class GitHubInfoResponse(
+    @SerializedName("full_name")
+    val fullName: String,
+
+    @SerializedName("description")
+    val description: String
+) {
+    fun toGitHubInfoModel(): GitHubRepositoryInfo {
+        return GitHubRepositoryInfo(
```

Comment:
변환하려는 타입과 함수명이 다르네요~
이 경우 반환되는 클래스명이 변경될 때마다 함수명이 변경되어야하는 이슈가 발생합니다!

```suggestion
    fun toDomainModel(): GitHubRepositoryInfo {
        return GitHubRepositoryInfo(
```

과 같은 형식으로 이용해보는 것은 어떨까요!?